sdk/azcore/request.go

```diff
@@ -187,6 +188,44 @@ func (req *Request) SetBody(body ReadSeekCloser, contentType string) error {
 	return nil
 }
 
+// WriteMultipartFormData writes the specified keys/values as multi-part form
+// fields with the specified value.  File content must be specified as a ReadSeekCloser.
+// All other values are treated as string values.
+func (req *Request) WriteMultipartFormData(formData map[string]interface{}) error {
```

Comment:
Maybe this should be `SetMultipartFormData()`?  Similar to `SetBody()`

Reply:
For consistency, I would say Set is better.

Reply:
Agreed, changed.